tests/plugins/plugin.text_selection.test.js

```diff
@@ -20,6 +20,7 @@ const FAKE_XML_MULT_WORDS = `<OBJECT data="file://localhost//tmp/derive/goodytwo
 <WORD coords="1600,2768,1700,2640">test3</WORD></LINE></PARAGRAPH></OBJECT>`;
 const FAKE_XML_5COORDS = `<OBJECT data="file://localhost//tmp/derive/goodytwoshoes00newyiala//goodytwoshoes00newyiala.djvu" height="3192" type="image/x.djvu" usemap="goodytwoshoes00newyiala_0001.djvu" width="2454">
 <PARAGRAPH><LINE><WORD coords="1216,2768,1256,2640,2690">test</WORD></LINE></PARAGRAPH></OBJECT>`;
+const FAKE_XML_EMPTY = '';
```

Comment:
So we get back an empty string instead of a 404 for missing dejavu files?

Reply:
yes. I took a look at the API server code and the logic is quite linear (+ python), it isn't a quick thing to sort out 😅  
But the smoking gun was empty XML file; this lil tweak handling the empty case will help us contain the non-render-blocking errors.